build/python311/patches/series

```diff
@@ -10,6 +10,7 @@ encoding-alias.patch
 locale-encoding.patch
 cgiserver.patch
 static-assert.patch
+pyc-timestamp.patch
```

Comment:
Was this patch already in the repository but not applied? I'm not seeing it in the file list.

Reply:
I missed the `git add` for the new file, sorry. Should be there now.